Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -2016,28 +2017,33 @@ static bool isContextMenuEvent(const WebMouseEvent& event)
 {
     m_lastMouseEvent = event;
 
+    // Even if the mouse event isn't handled (e.g. because the event is over a page we shouldn't
+    // display in Single Page mode), we should stop tracking selections (and soon autoscrolling) on MouseUp.
+    auto stopSelectionTrackingIfNeeded = makeScopeExit([this, isMouseUp = event.type() == WebEventType::MouseUp] {
+        if (isMouseUp)
+            stopTrackingSelection();
+    });
```

Comment:
How about this instead?
```
auto stopSelectionTrackingIfNeeded = makeScopeExit(event.type() == WebEventType::MouseUp
    ? [this] { stopTrackingSelection(); } : [this] { });
```

Reply:
I tried doing this, but the compiler gets confused by the expression and complains that the two operand types (both lambdas) are incompatible. Plus, `check-webkit-style` complains about having two statements in the same line, though that can be fixed with some new line wrapping.

Reply:
I'll keep it as-is for now!